environment/src/main/java/com/sequenceiq/environment/telemetry/service/AccountTelemetryService.java

```diff
@@ -190,4 +178,26 @@ private String createCRN(String accountId) {
                 .build()
                 .toString();
     }
+
+    private List<AnonymizationRule> loadAnonymizationRules() throws IOException {
```

Comment:
instead of reading the file manually, wouldn't be it better if we would use spring for this? like in `InstanceMetadataUpdater` we use it for reading the `packages` info.